pcsx2-qt/Settings/GameSummaryWidget.cpp

```diff
@@ -74,7 +70,15 @@ void GameSummaryWidget::populateDetails(const GameList::Entry* entry)
 	m_ui.crc->setText(QString::fromStdString(fmt::format("{:08X}", entry->crc)));
 	m_ui.type->setCurrentIndex(static_cast<int>(entry->type));
 	m_ui.region->setCurrentIndex(static_cast<int>(entry->region));
-	m_ui.compatibility->setCurrentIndex(static_cast<int>(entry->compatibility_rating));
+	m_ui.compatibility->setText(QString("%0%1")
+        .arg(GameList::EntryCompatibilityRatingToString(entry->compatibility_rating))
```

Comment:
Tabs should be used for indentation instead (my diffs tabs were turned to spaces by discord :face_exhaling: )